python/src/sdf/pyError.cc

```diff
@@ -79,7 +81,13 @@ void defineError(pybind11::object module)
         },
         "Sets the XML path that is associated with this error.")
```

Comment:
can you fix this doc-string while we are here? https://github.com/ignitionrobotics/sdformat/pull/933#discussion_r845628771